src/components/CriteriaProvider.tsx

```diff
@@ -89,7 +92,8 @@ export const CriteriaProvider: FC<CriteriaProviderProps> = memo(({ children }) =
       if (event.target?.result) {
         const rawElm = event.target.result as string;
         const newCriteria = jsonToCriteria(rawElm);
-        if (newCriteria) setCriteria(currentCriteria => [...currentCriteria, newCriteria]);
+        console.log(newCriteria);
```

Comment:
I think this was left here during debugging.

Reply:
Good catch - removed.